src/main/java/io/jenkins/stapler/idea/jelly/DeprecatedStaplerJellyCustomTagAttributeInspection.java

```diff
@@ -0,0 +1,23 @@
+package io.jenkins.stapler.idea.jelly;
+
+import com.intellij.codeInspection.LocalInspectionTool;
+import com.intellij.codeInspection.ProblemsHolder;
+import com.intellij.psi.PsiElementVisitor;
+import com.intellij.psi.XmlElementVisitor;
+import com.intellij.psi.xml.XmlAttribute;
+import org.jetbrains.annotations.NotNull;
+import org.kohsuke.stapler.idea.descriptor.StaplerCustomJellyTagfileXmlAttributeDescriptor;
+
+public class DeprecatedStaplerJellyCustomTagAttributeInspection extends LocalInspectionTool {
+    @Override
+    public @NotNull PsiElementVisitor buildVisitor(@NotNull ProblemsHolder holder, boolean isOnTheFly) {
+        return new XmlElementVisitor() {
+            @Override
+            public void visitXmlAttribute(@NotNull XmlAttribute attribute) {
+                if (attribute.getDescriptor() instanceof StaplerCustomJellyTagfileXmlAttributeDescriptor descriptor && descriptor.getModel().isDeprecated()) {
+                    holder.registerProblem(attribute, "Attribute is deprecated. Use \"Go to declaration\" to find the recommended solution.");
```

Comment:
is this needed? the docs clearly say what the solution is in your example, and that can be a convention instead?
```suggestion
                    holder.registerProblem(attribute, "Attribute is deprecated.");
```

Reply:
AFAIK, the convention is to have a quick-fix and if you don't have a quick-fix, then describe in more detail how to resolve the problem. https://jetbrains.design/intellij/text/inspections/#editor-messages Hence, I gravitated towards adding extra detail to the message.

My example is a mouse over the highlight, which shows the doc as well as the inspection. But the "Problems" Tool window does not show the doc. Also, when you ran inspections in bulk the doc does not show up either. All you get is "Attribute is deprecated...." without even a name of the attribute. 

I think I'll keep the long message, but make an update to include the name of the attribute.